src/components/page-layout/styles/layout.module.css

```diff
@@ -3,3 +3,12 @@
     max-width: 960;
     padding: 0 1.0875rem 1.45rem;
 }
+
+.modal{
```

Comment:
I'm not extremely comfortable with how css modules work so this may not be right, but. Since this is imported in `Layout.js`, won't that make `.modal` essentially a global style? So in that case we would want the class name here to be more specific - unless the goal here was actually to make a global style for the modals